### PR TITLE
Add support for scala 2.13

### LINKED ISSUE #12: Support Scala 2.13

### PR DESCRIPTION
**Pull request by taig**
Closes #12

### CODE REVIEW

**Review by julienrf — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -6,7 +6,7 @@ organization in ThisBuild := "org.julienrf"
 
 scalaVersion in ThisBuild := "2.13.0"
 
-crossScalaVersions := Seq("2.10.6", "2.11.12", "2.12.8", scalaVersion.value)
+crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.8", scalaVersion.value)
```
Comment (julienrf) [inline build.sbt:9]: Interesting. The tests fail on 2.11.12? Is that a known issue?

**Comment by julienrf** [comment]
Thanks!

**Comment by julienrf** [comment]
A release is on its way to maven central…

File: build.sbt (hunk 1 above)
Comment (taig) [reply build.sbt:9]: I didn't really bother to go down that rabbit hole. Perhaps a `clean` would have been sufficient.

**Pull request merged**